CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
-### 2.2.0 October 7, 2024
+### 2.3.0 October 7, 2024
 - rename wordpress-scripts to wordpres-utils in the README ([DESCW-2671](https://citz-gdx.atlassian.net/browse/DESCW-2671))
 
+### 2.1.3 May 2, 2024
+### 2.2.0 May 2, 2024
```

Comment:
i suspect it is not may, and also suspect we need to make sure we use the correct version, at this point the current version is 2.3.0, so it has to be something higher than this.